### PR TITLE
chore(ToastNotification) update storybook, increase test coverage

### LINKED ISSUE #220: Test coverage: Toast Notification
Coveralls indicates current coverage is 43% for Toast Notification. 

https://coveralls.io/github/patternfly/patternfly-react?branch=master

It would be nice to improve this.

Bonus: 
Our current Toast Notification example in Storybook  is not wrapped with a `toast-notifications-list-pf` div and alignment is off. Wrapping that example with `ToastNotificationList` should address the issue. 
screen shot 2018-02-13 at 1 08 35 pm

### PR DESCRIPTION
**Pull request by AllenBW**
fixes #220

yeah so here's that storybook thing, [look **LOOK** toast is no longer full view port 😏 ](https://allenbw.github.io/patternfly-react/?knob-Header=Great%20job%21&knob-Message=This%20is%20really%20working%20out.&knob-Type=success&knob-Dismiss=false&knob-Menu=true&knob-Action=true&selectedKind=ToastNotification&selectedStory=Toast%20Notification&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs)

~~close your eyes, don't look, list of things I KNOW gotta happen before this can be un-wipped (really we're just walking through the motions of throwing up a wip pr, gettin' da 👣 💧, secretly stealing travis runtime to get a sense of coverage increase )~~

* ~~need storybook~~
* ~~squash all da commits~~
* ~~even mo coverage~~
* ~~clean it all up~~
* ~~add some reviewers~~

### CODE REVIEW

**Comment by priley86** [comment]
test coverage up 0.4% - not sure if you are getting 100% on the toast notification now? you can check if you serve the `coverage/lcov-report/index.html` file built locally (and gitignored)... gives line by line analysis...

File: src/components/ToastNotification/ToastNotification.stories.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ stories.addWithInfo(
     const actionEnabled = boolean('Action', true);
 
     let story = (
-      <div>
+      <ToastNotificationList>
```
Comment (priley86) [inline src/components/ToastNotification/ToastNotification.stories.js]: fixes design alignment issue... just wanted folks to be aware of that...

**Comment by AllenBW** [comment]
So we're ready for a review! Don't hold back 😁

**Comment by priley86** [comment]
@AllenBW - beautiful 👏 👏 👏 ... any way to test the `TimedToastNotification` [paused timer](https://coveralls.io/builds/15562973/source?filename=src/components/ToastNotification/TimedToastNotification.js)? 

I am so thrilled about this... thank you!

**Comment by priley86** [comment]
@jgiardino can you confirm the new toast notification styling is correct? It seems much better to me 😄

**Comment by priley86** [comment]
p.s. @AllenBW i know this paused logic seems kind of obscure...but it is the current ux rule around hover.

**Comment by AllenBW** [comment]
@priley86 yeahhhhhh thats a gooooooood question... working that presently... i was hoping the mouse over after mount woulda paused it...

**Comment by priley86** [comment]
@AllenBW very nice work! this will help us a lot when trying to target tricky test scenarios in the future! 

🥇 💯 👍

**Pull request merged**